calicoctl/tests/st/utils/v1_data.py

```diff
@@ -194,26 +194,26 @@
     'spec': {'egress': [{'action': 'allow'}],
              'ingress': [{'action': 'deny',
                           'source': {'nets': ['10.0.20.0/24',
-                                              '192.168.000.000/32',
```

Comment:
The later versions of the net library seem to disallow leading 0s for IPv4 numbers.  Presumably to avoid confusion with octal.   It's a bit crap.